src/wp-includes/functions.php

```diff
@@ -8409,3 +8409,111 @@ function is_php_version_compatible( $required ) {
 function wp_fuzzy_number_match( $expected, $actual, $precision = 1 ) {
 	return abs( (float) $expected - (float) $actual ) <= $precision;
 }
+
+/**
+ **
+ * The number of active users in your installation.
+ *
+ * The count is cached and updated twice daily. This is not a live count.
+ *
+ * @since MU (3.0.0)
+ * @since 4.8.0 The `$network_id` parameter has been added.
+ * @since 6.0.0 Make to function.php.
+ *
+ * @param int|null $network_id ID of the network. Default is the current network.
+ * @return int Number of active users on the network.
+ */
+function get_user_count( $network_id = null ) {
+	if ( ! is_multisite() && null !== $network_id ) {
+		_doing_it_wrong( __FUNCTION__, __( 'Unable to pass $nework_id if not using multisite.' ), '6.0.0' );
+	}
+	return (int) get_network_option( $network_id, 'user_count', -1 );
```

Comment:
I think maybe I should revert this change.

Reply:
Prior to this change this function returns a numerical string despite its docblock saying `int`. I support casting it to an integer but I'm not sure what side effects that might have.

Reply:
We are now changing the default value to `-1`, meaning it will alway be an int anyway. Honestly, can't think of anything where this would not have been an int. In the multisite context, if this was false, then something went very wrong. Casting this to an int feels like a good change and it likely to fix more issues than it brakes. Not to mention it matches the doc block. 

I don't feel super strong about this through.

Reply:
Alright let's go with the int casting. A very cursory check with wpdirectory.net shows the top plugins assuming an integer or numeric return value: https://wpdirectory.net/search/01FYRSF0NRGQB1W3TAHY45578K